doc/classes/XRInterface.xml

```diff
@@ -256,7 +256,7 @@
 			Player is free to move around, full positional tracking.
 		</constant>
 		<constant name="XR_PLAY_AREA_STAGE" value="4" enum="PlayAreaMode">
-			Same as [constant XR_PLAY_AREA_ROOMSCALE] but origin point is fixed to the center of the physical space, [method XRServer.center_on_hmd] disabled.
+			Same as [constant XR_PLAY_AREA_ROOMSCALE] but origin point is fixed to the center of the physical space. In this mode, system-level recentering may be disabled, requiring the use of [method XRServer.center_on_hmd].
```

Comment:
I would suggest doing this the other way around. `XR_PLAY_AREA_ROOMSCALE` is the mode where the center of the playspace is locked in place as this is the default room scale behavior most headsets adhere to.

The new mode could then be called `XR_PLAY_AREA_ROOMSCALE_LOCAL` which then has a description such as `Same as XR_PLAY_AREA_ROOMSCALE` but the center of the play space is calibrate to the position of the player at startup, and when the player does a system recenter`.

Reply:
So, you're saying that `XR_REFERENCE_SPACE_TYPE_STAGE` should map to `XR_PLAY_AREA_ROOMSCALE`? We certainly could do that, the terminology can be whatever we want. :-)

However, it feels weird that `XR_PLAY_AREA_STAGE` would not equal `XR_REFERENCE_SPACE_TYPE_STAGE`, given that they both contain `STAGE` (and the project setting for this also calls it "Stage"). And given the original description of this constant (I only changed the end bit about `center_on_hmd()`), it certainly _seems_ like this value was meant to be meant to be used for `XR_REFERENCE_SPACE_TYPE_STAGE`.

Reply:
Owh, `XR_PLAY_AREA_STAGE` already exists... hmmm weird.. I see your point.. I think I really badly chose these names way back when :)

Reply:
I'm tempted to rename them to `XR_PLAY_AREA_ROOMSCALE_LOCAL` and `XR_PLAY_AREA_ROOMSCALE_STAGE`, just to set them apart as two roomscale variants..

We'll get in trouble with the production team for a breaking rename but I don't think anyone will even have been aware that we had these options seeing it was never implemented.

Reply:
If totally renaming them is a possibility, I think having just `XR_PLAY_AREA_LOCAL` and `XR_PLAY_AREA_LOCAL_FLOOR` would be great. Those do seem to be sort of standard names - they also exist in WebXR.

Do any low-level XR APIs even have a "space type" for "sitting" or "3dof"? Those are the other values on this enum, but they're not really mappable to anything in OpenXR or WebXR.

Reply:
I believe the mobile VR interface exposes itself as 3DOF as it doesn't have positional tracking, so I'd keep that. I'm happy with having that and the 3 suggestions you make, and removing the rest.

We'll have to make a clear statement as to why we're making this breaking change and why we think we can get away with it (this feature having been unimplemented is a strong indicator that nobody is currently using it:P ).

(obviously if we need to keep the old ones and give them the same const value and mark them as deprecated, that could be a way to make any critics happy, but I rather see them removed).